openapi/task_execution_service.openapi.yaml

```diff
@@ -741,6 +764,19 @@ components:
           example: 2020-10-02T10:00:00-05:00
           readOnly: true
       description: Task describes an instance of a task.
+    tesCallbackStatus:
+      type: object
+      required:
+      - task_id
+      - state
+      properties:
+        id:
```

Comment:
Inconsistent with `task_id` mentioned just above among the `required` properties. Change here or above.

Reply:
Yup, sorry I missed it :)